pkgs/misc/drivers/xp-pen-g430/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchzip, tar, autoPatchelfHook, libusb1, libX11, libXtst, qt5, libglvnd }:
+stdenv.mkDerivation rec {
+  name = "xp-pen-G430";
+  version = "20190820";
+
+  src = fetchzip {
+    url = "https://download01.xp-pen.com/file/2019/08/Linux%20Beta%20Driver(${version}).zip";
+    sha256 = "091kfqxxj90pdmwncgfl8ldi70pdhwryh3cls30654983m8cgnby";
+  };
+
+  nativeBuildInputs = [
+    tar
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    libusb1
+    libX11
+    libXtst
+    qt5.qtbase
+    libglvnd
+    stdenv.cc.cc.lib
+  ];
+
+  unpackPhase = ''
+    tar -xzf Linux_Pentablet_V1.3.0.0.tar.gz
```

Comment:
Probably can just do
```
src = fetchzip {
  url = "https://download01.xp-pen.com/file/2019/08/Linux%20Beta%20Driver(${version}).zip";
  sha256 = "091kfqxxj90pdmwncgfl8ldi70pdhwryh3cls30654983m8cgnby";
} + ./Linux_Pentablet_V1.3.0.0.tar.gz;
```
and drop the custom unpackPhase

Reply:
Was not aware this was possible, will do :+1: